src/swarm-api/swarm-utils.ts

```diff
@@ -0,0 +1,27 @@
+import { sendExtensionMessage } from '../messaging/e2e.messaging'
+/**
+ * Wrapper around sendExtensionMessage function.
+ * Checks whether the request was successful and converts outcome to Promise
+ * @param extensionId The Swarm extension ID
+ * @param action Action of the Swarm extension
+ * @param sessionId Session ID received by the Swarm extension
+ * @param parameters Depends of the action
+ * @returns Promise
+ */
+export async function sendSwarmExtensionMessage<Response>(
+  extensionId: string,
+  action: string,
```

Comment:
That's a good idea, but not that easy to do. The swarm-extension project must be configured to work as library and export types as well. So maybe it would be better to do that in a separate PR.